### PR TITLE
unit test for dashboardFooter

### PR DESCRIPTION
**Pull request by Dishebh**
For issue: #519

### CODE REVIEW

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-DashboardFooter/Components/index.test.tsx [hunk 1]
```diff
@@ -18,8 +18,13 @@ describe('<DashboardFooter />', () => {
...
   });
 
-  it('should render li, p tags', () => {
+  it('should render ul, li, p tags', () => {
+    expect(wrapper.find('ul').length).toBe(2);
     expect(wrapper.find('li').length).toBe(5);
     expect(wrapper.find('p').length).toBe(1);
   });
+
+  it('should render 2018 copywright text', () => {
```
Comment (Marvin9) [inline src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-DashboardFooter/Components/index.test.tsx:27]:
```suggestion
  it('should render 2018 copyright text', () => {
```

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-DashboardFooter/Components/index.test.tsx [hunk 2]
```diff
@@ -18,8 +18,13 @@ describe('<DashboardFooter />', () => {
...
+    expect(wrapper.find('p').text()).toEqual('© Ignitus 2018');
```
Comment (Marvin9) [inline src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-DashboardFooter/Components/index.test.tsx:28]: ```import { Copyright } from '../styles';```

```suggestion
    expect(wrapper.find(Copyright).text()).toEqual('© Ignitus 2018');
```

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-DashboardFooter/Components/index.test.tsx (hunk 1 above)
Comment (Dishebh) [reply src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-DashboardFooter/Components/index.test.tsx]: Thanks

File: src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-DashboardFooter/Components/index.test.tsx (hunk 2 above)
Comment (Dishebh) [reply src/ignitus-Shared/ignitus-DesignSystem/ignitus-Organisms/ignitus-DashboardFooter/Components/index.test.tsx]: Done

**Pull request merged**